app/src/main/java/com/mobileproto/hireddit/hireddit/MainActivity.java

```diff
@@ -1,41 +1,34 @@
 package com.mobileproto.hireddit.hireddit;
 
 import android.os.Bundle;
-import android.speech.tts.TextToSpeech;
-import android.support.design.widget.FloatingActionButton;
-import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
-import android.view.View;
 import android.view.Menu;
 import android.view.MenuItem;
 
-public class MainActivity extends AppCompatActivity
+public class MainActivity extends AppCompatActivity implements
+        MainFragment.OnFragmentInteractionListener, SpeakFragment.OnFragmentInteractionListener
 {
     public WordToSpeech speech;
 
+    FragmentManager manager;
+
+    public static final int HOME_SCREEN_CLICK = 1;
+
     @Override
     protected void onCreate(Bundle savedInstanceState)
     {
         super.onCreate(savedInstanceState);
+        manager = getSupportFragmentManager();
         setContentView(R.layout.activity_main);
+        switchFragment(MainFragment.newInstance());
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
```

Comment:
you can't really butterknife this line. `MainFragment.newInstance()' is very similar to

```
new MainFragment()
```